db/migrate/20190716121520_add_index_to_users_email.rb

```diff
@@ -0,0 +1,5 @@
+class AddIndexToUsersEmail < ActiveRecord::Migration[5.2]
+  def change
+    add_index :users, :email, unique: true
```

Comment:
@shaniajubelag
Let's not include this yet, since this is not related to this task.